Guidelines.md

```diff
@@ -298,7 +301,7 @@ For examples on use of OPTIONS, see [preflighting CORS cross-domain calls][cors-
 ### 7.5 Standard request headers
 The table of request headers below SHOULD be used by Microsoft REST API Guidelines services. Using these headers is not mandated, but if used they MUST be used consistently.
 
-All header values MUST follow the syntax rules set forth in the specification where the header field is defined. Many HTTP headers are defined in [RFC7231][rfc-7231], however a complete list of approved headers can be found in the [IANA Header Registry][IANA-headers]."  
+All header values MUST follow the rules set forth in [RFC2616][rfc-2616]; see "[token][rfc-2616-token]."  
```

Comment:
:exclamation: I believe this was correct before (it reverts a change made in #41)